lib/twitter_cldr/js/mustache/utilities.coffee

```diff
@@ -129,4 +129,14 @@ class TwitterCldr.Utilities
   @compute_cache_key : (pieces) ->
     if pieces? and pieces.length > 0
       return pieces.join("|")
-    return null
\ No newline at end of file
+    return null
+
+  @traverse_object : (obj, path) ->
+    value = obj
+    if path.length is 0
+      return null
+    for key in path
+      if value? and value.hasOwnProperty(key)
+        value = value[key]
+      else return null
```

Comment:
oops, that's how I usually do it. Something went wrong here. Fixed in the next commit